### ISSUE TITLE
'ps' command does not support '--format' option

### ISSUE THREAD

**Issue by jodh-intel** [01org/cc-oci-runtime#637]
- terminal 1 (start stand-alone `containerd`):

```
$ sudo systemctl stop docker
$ sudo containerd --version
containerd version 0.2.4
$ sudo containerd --debug
```

- terminal 2 (start a Clear Container):

```
$ sudo containerd-ctr --debug containers start --runtime /usr/bin/cc-oci-runtime.sh --attach wibble /path/to/bundle/dir/
```

- terminal 3 (attempt to list running containers):

```
$ sudo containerd-ctr containers list
[ctr] rpc error: code = 13 desc = get all pids for container: exit status 1: ""
$ sudo tail -2 /tmp/cor.log | cut -d: -f5-
cc-oci-runtime:debug:called as: /usr/bin/cc-oci-runtime ps --format=json wibble
cc-oci-runtime:warning:state file does not exist for container --format=json
```

The problem is that the `ps` command does not provide the `--format` option.

**Comment by jodh-intel** [comment]
Here's what `runc` provides:

```
runc ps --help
NAME:
   runc ps - ps displays the processes running inside a container

USAGE:
   runc ps [command options] <container-id> <ps options>

OPTIONS:
   --format value, -f value     select one of: table(default) or json


```

Until we fully support `ps`, a minimal fix would be to simply return `{}` for `ps --format=json $container` since that is a valid but empty JSON document.

**Comment by devimc** [comment]
Will be fixed in 3.0 - https://github.com/clearcontainers/runtime
